float64[] setup
---
float64[] solution
string info
bool success